broccoli/packages.js

```diff
@@ -355,7 +355,7 @@ module.exports.rollupEmberMetal = function _rollupEmberMetal(tree, options) {
     rollup: {
       input: `index.js`,
       output: {
-        moduleId: 'ember-metal',
```

Comment:
showing this `options.moduleId is deprecated in favour of options.amd = { id: moduleId }` in build time